index.js

```diff
@@ -0,0 +1,112 @@
+import { projects, navigationPoints } from "./constants.js";
+const burgerMenu = document.querySelector(".burger-menu");
+const headerNav = document.querySelector(".header__nav");
+const closeBtn = document.querySelector(".close-btn");
+const overlay = document.querySelector(".overlay");
+const container = document.querySelector(".cards__container");
+const navMenu = document.querySelector(".header__menu");
+const searchInput = document.querySelector(".search-box");
+let searchTimeout;
+
+function createNavigationMenu(navigationPoints) {
+  navMenu.innerHTML = "";
```

Comment:
I suggest putting variables to their corresponding functions
```suggestion
function createNavigationMenu(navigationPoints) {
  const navMenu = document.querySelector(".header__menu");
  navMenu.innerHTML = "";
```